Bottom: 3274db35d47c5c8ee3315b6b1f1e45d60db832f3
Top:    bc1b2817f7fa0fe78f0b632903bda018b5c46687
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-01-26 17:23:29 +0800

Refresh of debug

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index baf65cd..0527459 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -2096,7 +2096,7 @@ static int vhost_read_descs(struct vhost_virtqueue *vq, int num)
 	while (indices->tail < indices->head) {
 		printk("vq %p reading descs for indices tail %d\n",
 			vq, indices->tail);
-		head = vhost16_to_cpu(vq, indices->indices[indices->tail++]);
+		head = vhost16_to_cpu(vq, indices->indices[indices->tail]);
 		while(1) {
 			printk("vq %p read desc to descs head %d\n",
 				vq, descs->head);
@@ -2130,6 +2130,7 @@ static int vhost_read_descs(struct vhost_virtqueue *vq, int num)
 				goto err;
 			}
 		}
+		indices->tail++;
 	}
 
 done:
